### PR TITLE
duplicate part issue, prevent future duplicates.

### PR DESCRIPTION
**Pull request by davidvthecoder**
…licate partIDs for cart integration

### CODE REVIEW

File: models/cartIntegration/customerPricing.go [hunk 1]
```diff
@@ -35,12 +35,14 @@ var (
 		LEFT JOIN CartIntegration ci ON ci.partID = p.partID AND ci.custID = ?
 		left join Price pr on pr.partID = p.partID and pr.priceType = 'list'
 		WHERE p.status in (700, 800, 810, 815, 850, 870, 888, 900, 910, 950)  && p.brandID = ?
+		GROUP BY p.partID
```
Comment (dgallegos) [inline models/cartIntegration/customerPricing.go]: If we group by part.id does this mean someone can have multiple integrations with the same part Id?

Comment (dgallegos) [reply models/cartIntegration/customerPricing.go]: If someone has multipe p.partIds we need to clean up the data, I don't get the user case where we should allow this.

File: models/cartIntegration/fileio.go [hunk 2]
```diff
@@ -89,9 +89,9 @@ func UploadFile(file multipart.File, api_key string) error {
 			return err
 		}
 
-		custPartNum := cp.integrationExists(integrationLookup)
+		custPartNum, integExists := cp.integrationExists(integrationLookup)
 		if custPartNum != cp.CustomerPartID {
-			if custPartNum > 0 {
+			if integExists {
```
Comment (dgallegos) [inline models/cartIntegration/fileio.go]: I don't see how this logic changes anyting. now we're passing false when before we were passing 0. What are some example values, where the code you changed makes something different occur?

**Review by dgallegos — COMMENTED** [review]
I'm confused on something of these changes, can you explain them to me?

**Comment by dgallegos** [comment]
Talked to David. Decided to do following.

1. Remove Group By.
2. Clean Excel file coming in to use last partId if multiple partIds are in file.
3. Keep bool integrationExists that he added.

**Review by WileESpaghetti — APPROVED** [review]

**Pull request merged**